### ISSUE TITLE
Choice for playlists

### ISSUE THREAD

**Issue by 4anthony6** [Fabricio20/LewdWiki#125]
It would be cool if it was possible to control playlists by commands like:
**l!playlist** save_current [Myplaylist] : to save the current playlist with the name [Myplaylist].
**l!playlist** open [Myplaylist] : to open the playlist [Myplaylist]
**l!playlist** next : to play next playlist
**l!playlist** remove [Myplaylist] : to remove the playlist
**l!playlist** remove_all : to remove all the playlists
**l!playlist** list : to show a list of the playlists

(a lot of work... ahah)

**Comment by 4anthony6** [comment]
Cool! Can't wait to try that! 
You are right, maybe the commands I have listed are too long and complexes..

**Comment by Fabricio20** [comment]
Some things I'll probably be modifying:
- Open is a better alias than use (Currently it's **L!pl use <name>**).
- Remove_all is a nice command to have.
- Save_current could be an alias for the current "copy" command.

**Comment by 4anthony6** [comment]
I agree with your points!

Just a question, are these commands already been added to the wiki?

**Comment by Fabricio20** [comment]
Nope, they have not. I'm looking into wether the wiki here is actually useful (since I find it hard to maintain this here since it doesn't have a proper search function).

For now there are some references/docs over on https://notfab.net/LewdBot/docs

**Comment by 4anthony6** [comment]
Okay thank you! So I have added this link to my favs :)

**Comment by 4anthony6** [comment]
oh, and I think the delay for the timeout is too short for the l!setup commands...
